expyfun/_utils.py

```diff
@@ -765,10 +767,13 @@ def _wait_secs(secs, ec=None):
         while (clock() - t0) < secs:
             ec._dispatch_events()
             ec.check_force_quit()
+            time.sleep(0.0001)
```

Comment:
@rkmaddox maybe somehow this is the problematic change? It's easy enough to revert if need be.

Also do you do `ec.wait_secs` or `from expyfun.utils import wait_secs`? The former uses this code path, the latter uses the code path below.